### DISCUSSION TITLE
new site repo but old data repo

### DISCUSSION THREAD

**Discussion by WohnungNDS** [open-sdg/open-sdg#1475] [Q&A] [answered]
@LucyGwilliamAdmin @MoBosse 

Hi, 

in a couple of weeks I will get online with my Migration report. I have created a new github account ([christoph-LSN](https://github.com/christoph-LSN)) and forked the site starter an the data repo from opensgd. Then I have pointed in my [site_config](https://github.com/christoph-LSN/MT_Site/blob/develop/_data/site_config.yml) to my old [data repo](https://github.com/WohnungNDS/IM_Data)

On a first glance it works fine. Now I am wondering about to copy all my old data files to the new data repo? I have seen, that there is a major change in the meta data configuration. In my old data repo the meta data files containing all meta informations. I am not sure if I divide this files into the current structure of eg. 1.1.1.yml (meta) and 1.1.1.yml (indicator-config). What do you think?

**Comment by MoBosse** [comment]
@WohnungNDS Since I use the "old structure" with all the information in the meta files, I can't make any recommendations. In fact, I am not sure about the purpose of the division into meta and configuration files. With regard to "backwards-compatibility" both should work...

**Comment by brockfanning** [comment, marked as answer]
@WohnungNDS As you noticed the data starter currently starts with the metadata files and indicator configuration files both in YAML format, in separate folders. (More on the reasons for this below.) By contrast, your old data repo has the metadata and indicator configuration combined in a single folder of ".md" files.

It is possible to continue using the ".md" format, like in your old data repo, if you prefer. You would need to play around with the data configuration. Here is the relevant part of the data configuration controlling these inputs, in the current data starter: https://github.com/open-sdg/open-sdg-data-starter/blob/develop/config_data.yml#L45-L59

If you would like to continue using the .md format for metadata files, you could change `InputYamlMeta` to `InputYamlMdMeta`. You would also need to change `path_pattern: meta/*-*.yml` to `path_pattern: meta/*.md`. (I'm not sure why we have the double-asterisks in there.)

@MoBosse A bit about the reasons for splitting the indicator configuration out from the metadata. We created indicator configuration forms with the goal of making it easier to change indicator configuration - or at least less technical. These forms work by spitting out a downloadable file which has all of the indicator configuration. So, it was better to have indicator configuration live in its own file, so that it could be replaced directly, without requiring any edits. Meanwhile, the SDG SDMX working group is putting the final touches on a "metadata structure definition" - or in other words a global metadata schema. In order to be able to support this in Open SDG, it was also desirable to remove all of the Open-SDG-specific stuff (ie, indicator configuration) from the metadata files. So all of this led us to move towards splitting indicator configuration apart from metadata.

Lastly a technical note about the distinction between metadata and indicator configuration - as far as Open SDG is concerned, there is no actual distinction. You can see from the configuration linked above that both inputs use `InputYamlMeta`. Once everything is imported and being built, all of the indicator configuration and metadata are combined into one big key/value object. This is why it is perfectly fine if you want to continue using the .md files, combining metadata with configuration, like in your old data repo.

**Comment by christoph-LSN** [comment]
Thank you @brockfanning for this  full explanation! Because of some time pressure i try to use my old data repo for now and update to the version with the divided meta data files nex year
